app/controllers/developers_controller.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class DevelopersController < WyeworkersController
+  WyeworkerKind = Developer
+end
```

Comment:
No, this kind of meta programing should NEVER be used

Reply:
Got it, removed!